operator-framework-quarkus-extension/runtime/src/main/java/io/javaoperatorsdk/quarkus/extension/QuarkusControllerConfiguration.java

```diff
@@ -0,0 +1,126 @@
+package io.javaoperatorsdk.quarkus.extension;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.client.CustomResourceDoneable;
+import io.javaoperatorsdk.operator.api.config.ControllerConfiguration;
+import io.javaoperatorsdk.operator.api.config.RetryConfiguration;
+import io.quarkus.runtime.annotations.RecordableConstructor;
+import java.util.Collections;
+import java.util.Set;
+
+public class QuarkusControllerConfiguration<R extends CustomResource>
+    implements ControllerConfiguration<R> {
+  private final String name;
+  private final String crdName;
+  private final String finalizer;
+  private final boolean generationAware;
+  private final boolean clusterScoped;
+  private final Set<String> namespaces;
+  private final String crClass;
+  private final String doneableClassName;
+  private final boolean watchAllNamespaces;
+  private final RetryConfiguration retryConfiguration;
+
+  @RecordableConstructor
+  public QuarkusControllerConfiguration(
+      String name,
+      String crdName,
+      String finalizer,
+      boolean generationAware,
+      boolean clusterScoped,
+      Set<String> namespaces,
+      String crClass,
+      String doneableClassName,
+      RetryConfiguration retryConfiguration) {
+    this.name = name;
+    this.crdName = crdName;
+    this.finalizer = finalizer;
+    this.generationAware = generationAware;
+    this.clusterScoped = clusterScoped;
+    this.namespaces = namespaces;
+    this.crClass = crClass;
+    this.doneableClassName = doneableClassName;
+    this.watchAllNamespaces = this.namespaces.contains(WATCH_ALL_NAMESPACES_MARKER);
+    this.retryConfiguration =
```

Comment:
how can the user configure the retry in quarkus projects?

Reply:
I need to add the configuration part, indeed. I will create an issue for it.